Makefile

```diff
@@ -14,6 +14,7 @@ CPPFLAGS := -Iinclude $(CPPFLAGS)
 CFLAGS := -g -O2 -std=c99 -Wall -Werror -Wextra -Wpedantic -Wundef -Wconversion -Wno-missing-braces -fPIC -fvisibility=hidden $(CFLAGS)
 CC ?= cc
 AR ?= ar
+ARFLAGS ?= -r$(V0:1=v)
```

Comment:
According to #2716, the original `?=`s were once a workaround for #2716.
That issue’s root cause’s separately fixed and 7c2461f purged its related `extconf.rb` code. Like this PR, I wonder if those lines are still necessary?